assets/docs/fragments/cli-installation.md

```diff
@@ -0,0 +1,21 @@
+## CLI Installation
+<details>
+<summary>MacOS</summary>
+
+You can install in MacOS via brew: `brew install asyncapi`.
```

Comment:
if we can do collapse/expand then we can provide also other installation options, which in the case of MacOs it is also  `pkg` that we generate on each release -> https://github.com/asyncapi/cli#macos. Not all folks use `brew`